tests/integration/docusaurus/connecting_to_your_data/how_to_connect_to_sqlite_data.py

```diff
@@ -0,0 +1,80 @@
+import pathlib
+import sqlite3
+
+
+def get_sqlite_database_path() -> pathlib.Path:
+    relative_path = pathlib.Path(
+        "..",
+        "..",
+        "..",
+        "test_sets",
+        "taxi_yellow_tripdata_samples",
+        "sqlite",
+        "yellow_tripdata.db",
+    )
+    return pathlib.Path(__file__).parent.joinpath(relative_path).resolve(strict=True)
+
+
+sqlite_database_path = get_sqlite_database_path()
+assert sqlite_database_path.is_file()
+
+
+def get_table_name(sqlite_database_path: pathlib.Path) -> list:
+    conn = sqlite3.connect(sqlite_database_path)
+    c = conn.cursor()
+    c.execute("SELECT name FROM sqlite_schema WHERE type='table' ORDER BY name")
+    return c.fetchone()[0]
+
+
+my_table_name = get_table_name(sqlite_database_path)
+assert my_table_name is not None
+
+
+# <snippet name="tests/integration/docusaurus/connecting_to_your_data/how_to_connect_to_sqlite_data.py import">
+import great_expectations as gx
+
+context = gx.get_context()
+# </snippet>
+assert context is not None
```

Comment:
@TrangPham I am guessing that this assertion is probably not necessary.

Reply:
Will remove